smoketest/loggers.py

```diff
@@ -198,6 +198,21 @@ class _ShellLogger(Logger):
             "body: {7}",
             "",
         ]),
+        5:
```

Comment:
Would you mind just adding to level 4 instead of adding a 5th level? It'd keep the shell and JSON formats more similar, and I don't really think we need another level for this.